### PR TITLE
Allow no attributes to be set when specifying stores

### PR DESCRIPTION
**Pull request by spencerkimball**
Add unittest for store parsing.

[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/4329)

### CODE REVIEW

**Comment by mberhault** [comment]
LGTM

File: cli/flags.go [hunk 1]
```diff
@@ -146,6 +146,11 @@ attributes might also include speeds and other specs (7200rpm,
 200kiops, etc.). For example:`) + `
 
   --stores=hdd:7200rpm=/mnt/hda1,ssd=/mnt/ssd01,ssd=/mnt/ssd02,mem=1073741824
+
+` + wrapText(`
+If there are no device attributes, they can be omitted:`) + `
+
+  --stores=/mnt/ssd01,/mnt/ssd02
```
Comment (bdarnell) [inline cli/flags.go]: I would move this up above the form with attributes. I think store attributes are a relatively advanced feature that most people won't use, so we might as well simplify the common case. We can also remove the `ssd` attribute from the examples in the README.

Comment (spencerkimball) [inline cli/flags.go]: OK

**Comment by bdarnell** [comment]
LGTM

**Comment by jseldess** [comment]
Covered in docs for [`cockroach start`](https://www.cockroachlabs.com/docs/start-a-node.html).

**Pull request merged**